portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,85 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: 20px;
 }
+#nav_bar{
```

[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation): Use a space between the last selector and the declaration block.


```suggestion
#nav_bar {
```

Here and elsewhere.